packages/ui/src/components/Modal/footer.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Button } from '../Button'
+import { ModalProps } from './modal'
+
+type CondoFooterProps = Pick<ModalProps, 'footer' | 'okText' | 'cancelText' | 'onOk' | 'onCancel' | 'confirmLoading'>
+
+const ModalFooter: React.FC<CondoFooterProps> = (props) => {
+    const { footer, okText, cancelText, onOk, onCancel, confirmLoading } = props
+
+    if (footer === undefined) {
```

Comment:
it looks strange. If you have a custom footer why need to use this component?

Reply:
Removed `ModalFooter` component. Now there is no footer by default, and you can add a custom one